### DISCUSSION TITLE
Crud won't show up?

### DISCUSSION THREAD

**Discussion by vanvanni** [orchidsoftware/platform#1657] [General]
Hi,

Im trying to using the CRUD package but when I click the resources link in my nav it shows up blank or a http 500 code from my browser. I have cleared cache etc couple times but it never worked from the start, why? This is my current resource file:

```php
<?php

namespace App\Orchid\Resources;

use App\Models\Blog;
use Orchid\Crud\Resource;
use Orchid\Screen\Fields\Input;
use Orchid\Screen\TD;

class BlogResource extends Resource
{
    /**
     * The model the resource corresponds to.
     *
     * @var string
     */
    public static $model = Blog::class;

    /**
     * Get the fields displayed by the resource.
     *
     * @return array
     */
    public function fields(): array
    {
        return [
            Input::make('title')
                ->title('Title')
                ->placeholder('Enter title here')
        ];
    }

    /**
     * Get the columns displayed by the resource.
     *
     * @return TD[]
     */
    public function columns(): array
    {
        return [
            TD::make('id'),

            TD::make('created_at', 'Date of creation')
                ->render(function ($model) {
                    return $model->created_at->toDateTimeString();
                }),

            TD::make('updated_at', 'Update date')
                ->render(function ($model) {
                    return $model->updated_at->toDateTimeString();
                }),
        ];
    }

    /**
     * Get the filters available for the resource.
     *
     * @return array
     */
    public function filters(): array
    {
        return [];
    }
}

```

**Comment by tabuna** [comment]
You said you are getting 500 errors. What's the exception (message) there? 
I don't see a problem with your code.

**Reply by vanvanni** [reply]
> You said you are getting 500 errors. What's the exception (message) there?
> I don't see a problem with your code.

Thats the point. I see the stock browser 500 error with nothing in Laravel.log. Or anything else, I have no clue what the problem could be.

**Reply by tabuna** [reply]
In such a case, the script does not even get to the point of laravel execution. Try to look at the webserver logs

**Reply by vanvanni** [reply]
> In such a case, the script does not even get to the point of laravel execution. Try to look at the webserver logs

Fair point, nginx does log it within access.log but nothing within error.log. I will ook into this

**Reply by vanvanni** [reply]
It's because it somehow seems to arive at Laravel because if because if I change `domain.test/manage/crud/list/blog-resources` to `domain.test/manage/crud/blog-resources` I get the 404 page from Orchid. Only the list on the resources ends in http 500. And it should't be possible that just that route wont reach Laravel app

**Reply by vanvanni** [reply]
This caused the issue within the model:
```php
    protected static function boot()
    {
        self::creating(function ($model) {
            if($model->slug == null) {
                $model->slug = Str::slug($model->title);
            }
        });
        ```
